roundabout-core/src/test/java/at/fhv/itm3/s2/roundabout/entity/StreetSectionTest.java

```diff
@@ -15,18 +15,31 @@
 
 public class StreetSectionTest {
     @Test
-    public void firstCarCouldEnterNextSection() throws Exception {
+    public void firstCarCouldEnterNextSection_firstCarNotOnExitPoint() throws Exception {
         IStreetSection streetSectionMock = mock(StreetSection.class);
         when(streetSectionMock.firstCarCouldEnterNextSection()).thenCallRealMethod();
 
         // first car not on exit point
```

Comment:
redundant comment, same for the other methods, because you have used this in the method name now